09_employment/~compute_metrics_wage_ratio_2022.do.stswp

```diff
@@ -118,7 +118,7 @@ export delimited using metrics_wage_ratio_2022.csv, replace
 save "wage_ratio_final_2022.dta", replace
 
 ** add in county name and state name
-import delimited using "$crosswalk\county-populations.csv", stringcols(2 3 4 5) varnames(1) clear
+import delimited using "`crosswalk\county-populations.csv", stringcols(2 3 4 5) varnames(1) clear
```

Comment:
this is the last change I promise, everything else worked ok -- there is a quote missing at the end of crosswalk (should be `crosswalk')